package.json

```diff
@@ -20,32 +25,38 @@
     "lint": "eslint ./src/ --ext .ts",
     "lint:fix": "eslint ./src/ --ext .ts --fix",
     "check-types": "tsc --noEmit",
-    "audit": "npm audit"
+    "audit": "better-npm-audit audit"
   },
   "dependencies": {
-    "@makerxstudio/graphql-core": "^0.5.0",
-    "@makerxstudio/node-common": "0.1.0"
+    "@makerx/graphql-core": "^1.0.0",
+    "@makerx/node-common": "^1.0.3"
   },
   "peerDependencies": {
     "@apollo/server": "4 || 5",
     "graphql": "*"
   },
   "devDependencies": {
-    "@tsconfig/node16": "^1.0.3",
-    "@types/jest": "^29.2.3",
+    "@makerx/ts-config": "^1.0.1",
```

Comment:
Use `@makerx/ts-config` to avoid bundling issues. Will it be ok @cuzzlor ?
![image](https://github.com/MakerXStudio/graphql-apollo-server/assets/1898267/f1048d9d-a620-4a8a-b7c0-80a1829f60a5)

Reply:
Should be?